sys/pause.g

```diff
@@ -1,6 +1,10 @@
 ; pause.g - PAUSE CURRENT JOB
 
-; Raise the spindle to the top of
-; the Z axis and stop the spindle
-; but do not move the table.
+; Save pre-pause state of all general purpose
+; output pins.
+while { iterations < #state.gpOut }
+    set global.mosPS[iterations] = state.gpOut[iterations].pwm
+
```

Comment:
The states have been saved, but coolant options haven't been stopped.

Reply:
Thanks. I'm pretty sure this is handled in `G27` which is called straight after the pin states are saved, am I missing something? (there's a call to `M9` before the tool lifts away from the workpiece).